lib/src/test/java/io/ably/lib/test/realtime/RealtimeMessageTest.java

```diff
@@ -13,6 +13,7 @@
 import java.util.Locale;
 
 import com.google.gson.*;
+import io.ably.lib.types.*;
```

Comment:
I realise that the IDE did this on your behalf, but preferably keep PRs to just the changes that are the subject of the PR.